src/contacts/services/contacts.service.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  Injectable,
+  BadGatewayException,
+  BadRequestException,
+  Logger,
+} from '@nestjs/common'
+import { HttpService } from '@nestjs/axios'
+import { Campaign, PathToVictory } from '@prisma/client'
+import { lastValueFrom } from 'rxjs'
+import { ListContactsDTO } from '../schemas/listContacts.schema'
+
+type CampaignWithPathToVictory = Campaign & {
+  pathToVictory?: PathToVictory | null
+}
+
+const { PEOPLE_API_URL } = process.env
+
+@Injectable()
+export class ContactsService {
+  private readonly logger = new Logger(ContactsService.name)
+
+  constructor(private readonly httpService: HttpService) {}
+
+  async findContacts(
+    dto: ListContactsDTO,
+    campaign: CampaignWithPathToVictory,
+  ) {
+    if (!PEOPLE_API_URL) {
+      throw new BadGatewayException(
+        'PEOPLE_API_URL environment variable not configured',
+      )
+    }
```

Comment:
Maybe error should throw on startup if this is required functionality? If so, maybe different placement of existence check and error. If not, seems ok here.